tutorials/mnist_lr_tutorial.py

```diff
@@ -169,14 +165,17 @@ def print_privacy_guarantees(epochs, batch_size, samples, noise_multiplier):
        np.linspace(20, 100, num=81)])
   delta = 1e-5
   for p in (.5, .9, .99):
-    steps = math.ceil(steps_per_epoch * p)  # Steps in the last epoch.
-    coef = 2 * (noise_multiplier * batch_size)**-2 * (
```

Comment:
Not sure if first term in coef is correct here:
- The Lipshitz constant (using data_l2_norm) is missing.
- Why is batch_size here? If the RDP of Subsampled Gaussian Mechanism result [from table 1 of this work](https://arxiv.org/pdf/1908.10530.pdf) is being used, shouldn't `q = batch_size / steps_per_epoch` instead of `q = 1 / batch_size`?